woocommerce/woocommerce
# Update product template descriptions

### All Submissions:






### Changes proposed in this Pull Request:



Closes #33380.

This PR replaces the `"Pellentesque habitant.."` product descriptions in the product templates with more realistic examples.

 

### How to test the changes in this Pull Request:

1. Go to `add my products`
2. Click `Start with a template`
3. Choose `Physical product`
4. Observe that the `Hoodie in sweatshirt fabric made from a cotton blend.` text is shown in the product description.

### Other information:




### FOR PR REVIEWER ONLY:
